ts/private/ts_project_worker.js

```diff
@@ -84,6 +84,17 @@ function createFilesystemTree(root, inputs) {
         return node;
     }
 
+    function followSymlinkUsingRealFs(p) {
```

Comment:
Can we not use `realpath`?

Reply:
lstatSync is faster than realpath.

Reply:
do you mean instead of readLinkSync? if not;

lstatSync is fast so it makes sense to use it for distinguishing symlinks. the number of symlinks is a tiny portion of the inputs so it doesn't make sense to do a realpath for every input.

Reply:
Sounds good 👍 

Worth a comment so I don't ask the same question in the future? 😅

Reply:
Agreed.